src/index.html

```diff
@@ -18,5 +18,66 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="aimg"
+        href="dfrf"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol if they are meant to be internal page links.